src/pages/Cart/Cart.js

```diff
@@ -1,8 +1,150 @@
+      </div>
+      <main className="cartContainer">
+        <section className="cartProduct">
+          <div className="cartProductOption">
+            <div>
+              <input
+                type="checkbox"
+                onChange={e => handleAllCheck(e.target.checked)}
+                checked={checkItems.length === cartData.length}
+              />
+              <span>전체선택</span>
+              <span className="checkedItems">
+                ({checkItems.length}/{cartData.length})
+              </span>
+            </div>
+            <button
+              className="cartProductDeleteBtn"
+              onClick={() => {
+                if (checkItems.length !== 0) setIsModalOpen(true)
+              }}
+            >
+              선택삭제
+            </button>
+          </div>
+          {isModalOpen && (
+            <div className="modalLayout">
+              <div className="modalLayer" />
+              <Modal
+                isModalOpen={isModalOpen}
+                setIsModalOpen={setIsModalOpen}
+                deleteCartItem={deleteCartItem}
+              />
+            </div>
+          )}
+          <ul className="cartProductList">
+            {cartData.map((item, index) => {
+              return (
+                <li className="cartProductItem" key={index}>
+                  <input
+                    type="checkbox"
+                    onChange={e => handleCheck(e.target.checked, item.id)}
+                    checked={checkItems.includes(item.id)}
+                  />
+                  <img
+                    className="cartProductItemImg"
+                    src={item.url}
+                    alt={`${item.name}-product-img`}
+                  />
+                  <div className="cartProductName">{item.name}</div>
+                  <Count quantity={item.quantity} />
+                  <span className="cartProductPrice">
+                    {`${(item.price * item.quantity).toLocaleString()}원`}
+                  </span>
+                </li>
+              )
+            })}
+          </ul>
+        </section>
+        <section className="cartPurchase">
+          <ul className="cartPurchaseContainer">
+            <li className="cartPurchaseKeyAmount">
+              <span className="keyTitle">총 상품금액</span>
+              <span className="keyValue">{`${totalPrice.toLocaleString()}원`}</span>
+            </li>
+            <li className="cartPurchaseKeyAmount">
+              <span className="keyTitle">배송비</span>
+              <span className="keyValue">{`${deliveryPrice.toLocaleString()}원`}</span>
+            </li>
+            {totalPrice < 30000 ? (
+              <div className="deliveryAlert">
+                <div className="deliveryAlertMessage">{`${(
+                  30000 - totalPrice
+                ).toLocaleString()}원 더 주문 시, 무료배송`}</div>
+                <div className="deliveryAlertTriangle" />
+              </div>
+            ) : null}
```

Comment:
```suggestion
            {totalPrice < 30000 && (
              <div className="deliveryAlert">
                <div className="deliveryAlertMessage">{`${(
                  30000 - totalPrice
                ).toLocaleString()}원 더 주문 시, 무료배송`}</div>
                <div className="deliveryAlertTriangle" />
              </div>
            )}
```
조건에 부합하지 않는 경우가 있을 때는 && 연산자를 활용해주세요!